src/main/java/org/symphonyoss/integration/webhook/github/parser/v2/GithubCommentMetadataParser.java

```diff
@@ -0,0 +1,129 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.symphonyoss.integration.webhook.github.parser.v2;
+
+import static org.symphonyoss.integration.webhook.github.GithubEventConstants
+    .GITHUB_EVENT_COMMIT_COMMENT;
+import static org.symphonyoss.integration.webhook.github.GithubEventConstants
+    .GITHUB_EVENT_ISSUE_COMMENT;
+import static org.symphonyoss.integration.webhook.github.GithubEventTags.BODY_TAG;
+import static org.symphonyoss.integration.webhook.github.GithubEventTags.COMMENT_TAG;
+import static org.symphonyoss.integration.webhook.github.GithubEventTags.ENTITY_TAG;
+import static org.symphonyoss.integration.webhook.github.GithubEventTags.ENTITY_URL_TAG;
+import static org.symphonyoss.integration.webhook.github.GithubEventTags.EVENT_TAG;
+import static org.symphonyoss.integration.webhook.github.GithubEventTags.HTML_URL_TAG;
+import static org.symphonyoss.integration.webhook.github.GithubEventTags.ISSUE_TAG;
+import static org.symphonyoss.integration.webhook.github.GithubEventTags.USER_TAG;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.symphonyoss.integration.model.yaml.IntegrationProperties;
+import org.symphonyoss.integration.parser.ParserUtils;
+import org.symphonyoss.integration.parser.SafeString;
+import org.symphonyoss.integration.service.UserService;
+import org.symphonyoss.integration.webhook.github.parser.GithubParserUtils;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * This class is responsible to validate the event 'commit_comment' sent by Github Webhook when
+ * the Agent version is equal to or greater than '1.46.0'.
+ * Created by apimentel on 10/05/17.
+ */
+@Component
+public class GithubCommentMetadataParser extends GithubMetadataParser {
+
+  private static final String METADATA_FILE = "metadataGithubComment.xml";
+
+  private static final String TEMPLATE_FILE = "templateGithubComment.xml";
+  private static final String COMMIT_ENTITY = "commit";
+  private static final String ISSUE_ENTITY = "issue";
+
+  @Autowired
+  public GithubCommentMetadataParser(UserService userService, GithubParserUtils utils,
+      IntegrationProperties integrationProperties) {
+    super(userService, utils, integrationProperties);
+  }
+
+  @Override
+  protected String getTemplateFile() {
+    return TEMPLATE_FILE;
+  }
+
+  @Override
+  protected String getMetadataFile() {
+    return METADATA_FILE;
+  }
+
+  @Override
+  public List<String> getEvents() {
+    return Arrays.asList(GITHUB_EVENT_COMMIT_COMMENT, GITHUB_EVENT_ISSUE_COMMENT);
+  }
+
+  @Override
+  protected void preProcessInputData(JsonNode input) {
+    super.preProcessInputData(input);
```

Comment:
That's true because we don't have any common input data process in the super class, done.